packages/core/src/orders/redux/selectors.js

```diff
@@ -573,3 +573,42 @@ export const isOrderItemAvailableActivitiesLoading = state =>
  */
 export const getOrderItemAvailableActivitiesError = state =>
   getOrderItemAvailableActivities(state.orders).error;
+
+/**
+ * Returns the error for the order item available activities operations.
```

Comment:
I think you forgot to change the description 🙂

Reply:
Indeed, changed it 👍